commit fe7dadcfd59af3632302f807fad33e51f5b41be3 (HEAD -> master)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2019-06-11T00:15:56+00:00 (4 months ago)
    
    joyent/node-zkstream#48 doublecheck timer firing too often
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/lib/zk-session.js b/lib/zk-session.js
index eac1230..43a6f53 100644
--- a/lib/zk-session.js
+++ b/lib/zk-session.js
@@ -32,8 +32,8 @@ var METRIC_ZK_NOTIFICATION_COUNTER = 'zookeeper_notifications';
  * up to DOUBLECHECK_RAND. This spreads out the probes and avoids inducing
  * too much load on ZK.
  */
-var DOUBLECHECK_TIMEOUT = 900*1000;
-var DOUBLECHECK_RAND = 600*1000;
+var DOUBLECHECK_TIMEOUT = 4*3600*1000;
+var DOUBLECHECK_RAND = 8*3600*1000;
 
 function ZKSession(opts) {
 	mod_assert.object(opts, 'options');
diff --git a/package.json b/package.json
index 74de4d4..ed55fe3 100644
--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "zkstream",
-  "version": "0.11.7",
+  "version": "0.11.8",
   "description": "",
   "repository": {
     "type": "git",
